### PR TITLE
changed location of data being fetched to use the countries api

### PR DESCRIPTION
**Pull request by MrMaina100**
after review I saw you were fetching data from the data.json file which frontend-Mentor was providing you ( I assume)  with but the data is outdated. Some countries had outdated names while other countries had a population of zero.

### CODE REVIEW

**Comment by iamkiprono** [comment]
Can you re factor the catch block to display error on the screen instead of logging it to the console, that is incase of server time out

File: src/Components/Homepage.js [hunk 1]
```diff
@@ -6,9 +6,14 @@ const Homepage = () => {
...
 const getCountries = async () => {
-  const res = await fetch("data.json")
-  const data = await res.json()
-  setCountries(data)
+   try {
+      const res = await fetch("https://restcountries.com/v3.1/all");
+      const data = await res.json();
+      setCountries(data);
+    } catch (error) {
+      console.log("Error fetching countries:", error);
```
Comment (iamkiprono) [inline src/Components/Homepage.js]: try to set error to the screen instead of logging to console

Comment (MrMaina100) [reply src/Components/Homepage.js]: will do that

**Comment by MrMaina100** [comment]
added the custom error...

**Comment by iamkiprono** [comment]
Looks good to me

**Pull request merged**